### PR TITLE
* Added YouTube video on configuration page * Formatted configuration page texts for all supported languages * Removed unused files

### PR DESCRIPTION
**Pull request by FerhanGit**

### CODE REVIEW

File: views/templates/admin/configuration.tpl [hunk 1]
```diff
@@ -43,11 +44,12 @@
             	{/if}
             	
             {/foreach}
-    {else}
-
-    {/if}
+   
 	</ul>
 </div>
+{else}
```
Comment (Quetzacoalt91) [inline views/templates/admin/configuration.tpl]: Unused `{else}` which can be safely removed

File: views/templates/admin/setup_landing.tpl [hunk 2]
```diff
@@ -24,9 +24,19 @@
...
     <div class="logo_mailjet_center"></div>
-    {$MJ_templates.setup_landing_message|default:''}
-
-    <div id="setup_landing_bt">
+   
+    <ul style="list-style: none; margin-top:50px;">
+        <li id="landingMsg" style="display: inline;float:left; width:450px;">    
+        {$MJ_templates.setup_landing_message|default:''}
+        </li>
+        <li id="divYouTubePlayer" style="display: inline; float:left; width:400px; margin-left:50px;">
```
Comment (Quetzacoalt91) [inline views/templates/admin/setup_landing.tpl]: Your style inline should be moved in CSS files

File: views/templates/admin/setup_landing.tpl [hunk 3]
```diff
@@ -24,9 +24,19 @@
 *}
 <div class="center_page">
     <div class="logo_mailjet_center"></div>
-    {$MJ_templates.setup_landing_message|default:''}
-
-    <div id="setup_landing_bt">
+   
+    <ul style="list-style: none; margin-top:50px;">
+        <li id="landingMsg" style="display: inline;float:left; width:450px;">
```
Comment (Quetzacoalt91) [inline views/templates/admin/setup_landing.tpl]: Your style inline should be moved in CSS files.

**Comment by Quetzacoalt91** [comment]
Hi @FerhanGit,

Can you please update the version in mailjet.php and config.xml ? Thanks

**Comment by FerhanGit** [comment]
@Quetzacoalt91 
Thank you for the review - all required modofications are done and PR updated.

**Comment by Quetzacoalt91** [comment]
Hi @FerhanGit,

Thanks, I have just merged your PR had the changes will be available from tomorrow.

Best regards

**Pull request merged**